src/wp-includes/revision.php

```diff
@@ -1131,3 +1131,32 @@ function _wp_preview_meta_filter( $value, $object_id, $meta_key, $single ) {
 
 	return get_post_meta( $preview->ID, $meta_key, $single );
 }
+
+/**
+ * Add any revisioned meta fields to the revisions screen.
+ *
+ * @since 6.4.0
+ */
+function wp_add_revisioned_meta_to_revision_ui( $fields, $post ) {
```

Comment:
Missing @param document for the new function.

Reply:
added